### ISSUE TITLE
How to Address NaN Values and Loss Divergence Issues During Training?

### ISSUE THREAD

**Issue by Square-of-W** [revsic/torch-diffusion-wavegan#2]
When attempting to replicate the repository on the LJ Speech dataset following the given steps, I encountered issues with loss function divergence. Are there any specific techniques or strategies to tackle this instability during training?
![image](https://github.com/revsic/torch-diffusion-wavegan/assets/51012450/f2c627d8-25b3-4709-9e6c-9af6e9bff950)
